docs/io/visualization/sdec_plot.ipynb

```diff
@@ -13,8 +13,8 @@
    "execution_count": null,
```

Comment:
<p>Correction: "...first value is the wavelength and..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1761/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>